test/runnable/selectiveUnittests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+unittest_output="unittesting printingUnittest"
+
+$DMD -m${MODEL} -I=${EXTRA_FILES} -unittest ${EXTRA_FILES}/printingUnittest.d -run ${EXTRA_FILES}/selectiveUnittests.d | grep -q "${unittest_output}"
+
+obj_file=${OUTPUT_BASE}/temp${OBJ}
+
+# Precompile the module and make sure it's unittest doesn't get run
+$DMD -m${MODEL} -c -of=${obj_file} ${EXTRA_FILES}/printingUnittest.d
+$DMD -m${MODEL} -I=${EXTRA_FILES} -unittest ${obj_file} -run ${EXTRA_FILES}/selectiveUnittests.d | grep -q -v "${unittest_output}"
```

Comment:
Maybe add something along the lines of:

compilable/extra-files/minimal/verify_symbols.sh
compilable/extra-files/test11237.sh
runnable/test17619.sh

Just grep for `__unittest` in the output of `nm` or `objdump`